### PR TITLE
Fixes #158 : Get messages by roomId

### LINKED ISSUE #158: [back] get messages by roomId
Backend doesn't implement a way to fetch all messages of a specified room using it's ID :+1: 
```
curl -v http:/localhost:5000/api/message?roomid=6043978fd68dc3fcbf6079d6
```

This is required for the front interface.

### PR DESCRIPTION
**Pull request by Draeggon**
fix #158

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -171,5 +171,5 @@ curl -X DELETE http://localhost:5000/api/message/0
 
 Get all messages for one RoomId:
 ```sh
-curl -v http:/localhost:5000/api/message?roomid=6043978fd68dc3fcbf6079d6 # doesn't work here but it doesn't worked before need to open an issue
+curl -v http:/localhost:5000/api/message?roomId=6043978fd68dc3fcbf6079d6 # doesn't work here but it doesn't worked before need to open an issue
```
Comment (yannis-mlgrn) [inline server/README.md:174]: remove comments

File: server/scratchy_server/resources/messageres.py [hunk 2]
```diff
@@ -28,9 +23,15 @@ def delete(self, messageId):
...
 
 
+@doc(tags=['Message'])
+@marshal_with(AllMessageSchema)
 class NoIdMessageRes(MethodResource):
 
-    @doc(tags=['Message'])
+    @use_kwargs({"roomId": fields.String()}, location="query")
+    def get(self, roomId):
+        print(roomId)
```
Comment (Benvii) [inline server/scratchy_server/resources/messageres.py:32]: Remove unused `print(roomId)`

**Review by yannis-mlgrn — CHANGES_REQUESTED** [review]
Please add comments for everyone understand your code

File: server/scratchy_server/test/test_AllMessage.tavern.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  - *postMessage
+
+  - name: Get all messages in a room
+    request:
+      url: "{url}/message?roomId={24-zero}"
+      method: GET
+    response:
+      json:
+        messages:
+          - id: !re_fullmatch "[0-9a-f]{24}"  # the regex check if id is an objectId
```
Comment (Benvii) [inline server/scratchy_server/test/test_AllMessage.tavern.yaml:39]: Take care of the tests order might impact the messages results.

File: server/README.md (hunk 1 above)
Comment (Benvii) [inline server/README.md:174]: Remove the comment `# doesn't work here but it doesn't worked before need to open an issue`

**Review by Benvii — CHANGES_REQUESTED** [review]

**Comment by Benvii** [comment]
Made in #195

**Pull request closed without merging**